onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -489,14 +521,8 @@ void OpTester::Run(ExpectResult expect_result,
     so.enable_sequential_execution = sequential_execution;
 
     static const std::string all_provider_types[] = {
-        kCpuExecutionProvider,
-        kCudaExecutionProvider,
-        kMklDnnExecutionProvider,
-        kNGraphExecutionProvider,
-        kNupharExecutionProvider,
-        kBrainSliceExecutionProvider,
-        kTensorrtExecutionProvider,
-        kOpenVINOExecutionProvider,
+        kCpuExecutionProvider,    kCudaExecutionProvider,       kMklDnnExecutionProvider,   kNGraphExecutionProvider,
+        kNupharExecutionProvider, kBrainSliceExecutionProvider, kTensorrtExecutionProvider, kOpenVINOExecutionProvider,
     };
```

Comment:
nit: Is this default clang-format formatting?

Reply:
Fixed.

Reply:
Sorry about this.